src/objects/deck/demo/articles.twig

```diff
@@ -1,7 +1,7 @@
 {% if columns and columns > 1 and columnsBreakpoint and columnsBreakpoint != 'none' %}
-  {% set _deck_class = 'o-deck--' ~ columns ~ '-column' ~ columnsBreakpoint %}
+  {% set _deck_class = 'o-deck--' ~ columns ~ '-column' ~ columnsBreakpoint ~ ' ' ~ class ~ ' ' ~ alignment %}
 {% else %}
-  {% set _deck_class = '' %}
+  {% set _deck_class = class ~ ' ' ~ alignment %}
```

Comment:
We had a class arg already but it wasn't being rendered. I added it and the new alignment class arg.